### PR TITLE
Super-fy the zq command doc

### PR DESCRIPTION
**Pull request by philrz**
If reviewers would like to just read the content in rendered form, I've published it to a personal staging site. The key pages:

* https://spiffy-gnome-8f2834.netlify.app/docs/next/commands/super
* https://spiffy-gnome-8f2834.netlify.app/docs/next/commands

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ questions.
 * Rework dataflow model and Zed compiler optimizations (#2669)
 * Add initial `explode` operator that can break values from complex fields out into separate records (#2673)
 * Fix an issue where including a particular `time`-typed field in a shaper script caused errors with shaping other fields (#2685)
-* Silently discard duplicate fields when reading NDJSON records, which works around [Suricata bug 4016](https://redmine.openinfosecfoundation.org/issues/4106) (#2691)
+* Silently discard duplicate fields when reading NDJSON records, which works around Suricata bug 4016 (#2691)
```
Comment (philrz) [inline CHANGELOG.md:438]: This link to Suricata's flaky bug tracking system kept angering the broken link checker so I just moved the hyperlink into the referenced super issue #2691 and dropped it from this CHANGELOG.

File: docs/commands/super.md [hunk 2]
```diff
@@ -1,83 +1,75 @@
...
-# zq
+# `super`
 
-> **TL;DR** `zq` is a command-line tool that uses the [Zed language](../language/README.md)
-for pipeline-style search and analytics.  `zq` can query a variety
-of data formats in files, over HTTP, or in [S3](../integrations/amazon-s3.md) storage.
-It is particularly fast when operating on data in the [Super Binary](../formats/bsup.md) format.
+> **TL;DR** `super` is a command-line tool that uses [SuperSQL](../language/README.md)
+> to query a variety of data formats in files, over HTTP, or in [S3](../integrations/amazon-s3.md)
+> storage. It is particularly fast when operating on data in binary formats such as
```
Comment (mccanne) [inline docs/commands/super.md:10]: Let's rephrase this as context is different now...

**Review by mccanne — APPROVED** [review]
